### PR TITLE
Feature/docs 1895 get rid of wallarm api security

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Wallarm Overview
 
-Wallarm API Security is uniquely suited to protect your cloud applications and APIs. Its hybrid architecture safeguards your resources by offering:
+Wallarm API Security Platform is uniquely suited to protect your cloud applications and APIs. Its hybrid architecture safeguards your resources by offering:
```
Comment (AnastasiaTWW) [inline README.md:3]: Platform - should not be capitalized

File: docs/latest/about-wallarm/api-abuse-prevention.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # API Abuse Prevention
 
-The **API Abuse Prevention** module of Wallarm API Security delivers detection and mitigation of bots performing API abuse like credential stuffing, fake account creation, content scraping and other malicious actions targeted at your APIs.
+The **API Abuse Prevention** module of Wallarm platform delivers detection and mitigation of bots performing API abuse like credential stuffing, fake account creation, content scraping and other malicious actions targeted at your APIs.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-abuse-prevention.md:3]: the Wallarm platform?

File: docs/latest/about-wallarm/deployment-best-practices.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Wallarm API Security deployment and maintenance best practices
+# Wallarm solution deployment and maintenance best practices
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/deployment-best-practices.md:1]: The Wallarm solution?

File: docs/latest/admin-en/installation-guides/envoy/envoy-docker.md [hunk 4]
```diff
@@ -6,11 +6,11 @@
 
 These instructions describe the steps to run the Wallarm Docker image based on [Envoy 1.18.4](https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.18.4). The image contains all systems required for correct Wallarm node operation:
 
-* Envoy proxy services with embedded Wallarm API Security module
+* Envoy proxy services with embedded Wallarm module
 * Tarantool modules for postanalytics
 * Other services and scripts
 
-Wallarm API Security module is designed as an Envoy HTTP filter for requests proxying.
+Wallarm module is designed as an Envoy HTTP filter for requests proxying.
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/installation-guides/envoy/envoy-docker.md:13]: The Wallarm module?

File: docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ There are 4 examples representing frequent deployment approaches:
 
 [This example](https://github.com/wallarm/terraform-aws-wallarm/tree/main/examples/proxy) demonstrates how to deploy Wallarm as an inline proxy to AWS Virtual Private Cloud (VPC) using the Terraform module.
 
-Wallarm proxy solution provides an additional functional network layer serving as an advanced HTTP traffic router with the WAF and API security functions. This is the **recommended** deployment option since it provides the most functional and easy to implement solution.
+Wallarm proxy solution provides an additional functional network layer serving as an advanced HTTP traffic router with the application protection and API security functions. This is the **recommended** deployment option since it provides the most functional and easy to implement solution.
```
Comment (AnastasiaTWW) [inline docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md:16]: application protection -> WAAP

File: docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md [hunk 6]
```diff
@@ -30,15 +30,15 @@ You can see the solution flexibility in action by trying the [proxy advanced sol
 
 [This example](https://github.com/wallarm/terraform-aws-wallarm/tree/main/examples/advanced) demonstrates how to deploy Wallarm as an inline proxy with advanced settings to AWS Virtual Private Cloud (VPC) using the Terraform module. It is a lot like the [simple proxy deployment](#proxy-solution) but with some frequent advanced configuration options demonstrated.
 
-Wallarm proxy advanced solution (as well as a simple proxy) provides an additional functional network layer serving as an advanced HTTP traffic router with the WAF and API security functions.
+Wallarm proxy advanced solution (as well as a simple proxy) provides an additional functional network layer serving as an advanced HTTP traffic router with the application protection and API security functions.
```
Comment (AnastasiaTWW) [inline docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md:33]: WAAP

File: docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md [hunk 7]
```diff
@@ -30,15 +30,15 @@ You can see the solution flexibility in action by trying the [proxy advanced sol
...
 
 [Refer to the example deployment guide on GitHub](https://github.com/wallarm/terraform-aws-wallarm/tree/main/examples/advanced)
 
 ## Proxy solution for Amazon API Gateway
 
 [This example](https://github.com/wallarm/terraform-aws-wallarm/tree/main/examples/apigateway) demonstrates how to protect [Amazon API Gateway](https://aws.amazon.com/api-gateway/) with Wallarm deployed as an inline proxy to AWS Virtual Private Cloud (VPC) using the Terraform module.
 
-Wallarm proxy solution provides an additional functional network layer serving as an advanced HTTP traffic router with the WAF and API security functions. It can route requests to almost any service type including Amazon API Gateway without limiting its capabilities.
+Wallarm proxy solution provides an additional functional network layer serving as an advanced HTTP traffic router with the application protection and API security functions. It can route requests to almost any service type including Amazon API Gateway without limiting its capabilities.
```
Comment (AnastasiaTWW) [inline docs/latest/installation/cloud-platforms/aws/terraform-module/examples.md:41]: Next-Gen WAF

File: docs/latest/installation/kubernetes/kong-ingress-controller/customization.md [hunk 8]
```diff
@@ -8,30 +8,30 @@ Kong Ingress Controller with integrated Wallarm services is based on the standar
...
-* Via the Wallarm Console UI - it allows fine-tuning the Wallarm API Security settings.
+* Via the Wallarm Console UI - it allows fine-tuning the Wallarm settings.
 
 ## Configuration of Kong API Gateway
 
 Configuration of Kong Ingress Controller for Kong API Gateway is set by the [default Helm chart values](https://github.com/wallarm/kong-charts/blob/main/charts/kong/values.yaml). This configuration can be overridden by the `values.yaml` file provided by the user during `helm install` or `helm upgrade`.
 
 To customize the default Helm chart values, learn the [official instructions on the Kong and Ingress Controller configuration](https://github.com/Kong/charts/tree/main/charts/kong#configuration).
 
-## Configuration of the Wallarm API Security layer
```
Comment (AnastasiaTWW) [inline docs/latest/installation/kubernetes/kong-ingress-controller/customization.md:24]: as for changes in headers - please check if other documents refer to headers with old names

Comment (MaximBashurov) [reply docs/latest/installation/kubernetes/kong-ingress-controller/customization.md:24]: Will be done as a separate commit.

Comment (MaximBashurov) [reply docs/latest/installation/kubernetes/kong-ingress-controller/customization.md:24]: Fixed

**Pull request merged**